content/blog/2023/2023-q1-update/index.md

```diff
@@ -119,6 +119,8 @@ We responded with some of our ideas around how to build more open, inclusive, co
 Our [Open Engineering Team](https://2i2c.org/organization/#faces) continues to grow!
 We added two new team members to help improve and manage our cloud infrastructure service: [Erik Sundell](https://2i2c.org/author/erik-sundell/) and [Pris Nasrat](https://2i2c.org/author/pris-nasrat/).
 
+> **Note**: Pris left our team in July of 2023. We appreciate the work that she did with us and wish her the best in her next endeavors.
```

Comment:
Personally I don't think it's necessary to include me in the post at all, in my experience generally companies don't post new hires/departures. 

Note as you are removing me from the website in this PR the link above will break also.  I'd suggest just added Erik and leave as that. The history is in git commit.

Reply:
OK, I'll go with this suggestion, thanks